src/com/lftechnology/java/training/srijan/basics/coreapp/dao/DaoFactory.java

```diff
@@ -0,0 +1,8 @@
+package com.lftechnology.java.training.srijan.basics.coreapp.dao;
+
+public class DaoFactory {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)